hardware/MySensorsBase.cpp

```diff
@@ -1169,11 +1170,13 @@ void MySensorsBase::UpdateSwitch(const _eSetType vType, const unsigned char Idx,
 		}
 	}
 
-	//Send as Lighting 2
+	// LLEMARINEL : #1312  Changed to use as pTypeGeneralSwitch
+	// --- Send as Lighting 2
 	tRBUF lcmd;
 	memset(&lcmd, 0, sizeof(RBUF));
 	lcmd.LIGHTING2.packetlength = sizeof(lcmd.LIGHTING2) - 1;
-	lcmd.LIGHTING2.packettype = pTypeLighting2;
+	lcmd.LIGHTING2.packettype = pTypeGeneralSwitch;
+	//lcmd.LIGHTING2.packettype = pTypeLighting2;
 	lcmd.LIGHTING2.subtype = sTypeAC;
```

Comment:
pTypeGeneralSwitch should not be send like this, see for an example RFLinkBase.cpp
CRFLinkBase::SendSwitchInt
You need to use _tGeneralSwitch